no-computer.md

```diff
@@ -1,55 +1,68 @@
 
 ### Why you?
 _What excites you about this idea? How did you come up with it?_
 
+I LOVE baseball and this is totally something I would use! I think baseball, which is one of the stats-heaviest sports I can think of, lends itself well to computation and simulation well.
 
 ### Domain
 _Describe the project's domain in five words._
 
+Helping people simulate baseball events
 
 ### Interface (syntax)
 _How might the user interact with the language? What does programming look 
 like? Why is this the right way to interact with the problem domain?_ 
 
+I think the user would input either existing players (whose hitting zones, pitch preferences etc would come directly from the internet) or create players within the program, as well as a pitcher with certain pitches, pitch speed, and so on, and then ask at what likelihood different events would occur (single, strikeout etc) if a pitcher through a certain pitch aimed for a certain area. This is the right way to interact with the domain because the user is given the ability to specify exactly the desired event.
+
+The DSL is massively extensible, which is both a blessing and a curse. The field itself can be broken up into (arbitrarily small) zones and then, if a physics engine is applied, we could actually simulate where a ball would go, with what exit velocity, and so on.
 
 ### Operation (semantics)
 _What might happen when a program runs? How does a program interact with the
 user? What kinds of errors might occur, and how might they be communicated to
 the user?_
 
+I think at first, the program could look up how hitters have dealt with certain pitches in certain zones in the past, and also see how well pitchers have successfully located pitches in the desired location in the past (baseball has lots and lots of data -- this would be easy to look up/reference), in order to provide some sort of probability of various outcomes. Eventually a physics simulation could be added, which would actually look at contact with the bat and provide a prediction for where a ball would land.
+
+Errors would possibly include insufficient data (at which point perhaps the program could harbor a bad guess or something like that), or if the user inputs unreasonable inputs, which the program wouldn't be able to do anything with, such as the pitcher throwing 200MPH or something like that. I suppose the program would simply pop up an alert/error
 
 ### Expressiveness
 _What should be easy to do in this language? What should be possible, but
 difficult? What should be impossible or very difficult?_
 
+It should be easy to make predictions as to how batters will do against certain pitchers (as this is the intended functionality). It should be difficult to ask for unrealistic projections, such as impossibly fast pitchers. It should be impossible to create non-standard baseball outcomes, like a ball hitting a bird or having fan interference disrupt play.
```

Comment:
I wonder if baseball managers use stuff like this when deciding pay and trades? I feel like they must do some type of simulations since ever roster choice is a multi-million dollar decision.